src/__smoke-test__/smoke-test.spec.ts

```diff
@@ -0,0 +1,446 @@
+        eventList.push(parsedPayload[i].id);
+    }
+}
+
+function getEventsByType(requestBody, eventType) {
+    return requestBody.filter((e) => e.type === eventType);
+}
+
+function filterSuccessfulResponse(response, requestBody) {
+    // filter out irrelevant, unsuccessful network request/response
+    const request = response.request();
+    if (
+        request &&
+        response.status() === 200 &&
+        request.method() === 'POST' &&
+        response.url() === endpoint + monitorId + '/' &&
+        request.postData()
+    ) {
+        requestBody.push(request.postData());
+    }
+}
+
+// Timestamp to filter events fetched via getAppMonitorData API
+const time = Date.now();
+
+test.describe('Smoke test fixtures', () => {
+    let page;
+    const eventIds = [];
+
+    // Use single browser context to prevent multiple sessions per test
+    test.beforeAll(async ({ browser }) => {
+        const context = await browser.newContext();
+        page = await context.newPage();
+        await page.goto(url);
+    });
+
+    test.afterAll(async ({ browser }) => {
+        browser.close;
+    });
+
+    test('test initial events are emitted', async () => {
+        const SESSION_START_COUNT = 1;
+        const PAGEVIEW_COUNT = 1;
+        const NAVIGATION_COUNT = 1;
+        const RESOURCE_EVENT_COUNT = 1;
+        const expectedPage = '/smoke.html';
+        const requestBody = [];
+
+        // Collect successful requests
+        page.on('response', (response) =>
+            filterSuccessfulResponse(response, requestBody)
+        );
+
+        // Wait until fetch is complete
+        await page.waitForTimeout(10000);
+
+        // Parse payload
+        const parsedPayload = parseBatches(requestBody);
+        const session = getEventsByType(
+            parsedPayload,
+            SESSION_START_EVENT_TYPE
+        );
+        const navigation = getEventsByType(
+            parsedPayload,
+            PERFORMANCE_NAVIGATION_EVENT_TYPE
+        );
+        const pageView = getEventsByType(parsedPayload, PAGE_VIEW_EVENT_TYPE);
+        const resource = getEventsByType(
+            parsedPayload,
+            PERFORMANCE_RESOURCE_EVENT_TYPE
+        );
+
+        // Verify action is captured
+        expect(session.length).toEqual(SESSION_START_COUNT);
+        expect(pageView.length).toEqual(PAGEVIEW_COUNT);
+        expect(navigation.length).toEqual(NAVIGATION_COUNT);
+        expect(resource.length).toEqual(RESOURCE_EVENT_COUNT);
+
+        // Parse metadata to verify pageId
+        const pageId = JSON.parse(pageView[0].metadata).pageId;
+        const navigationPageId = JSON.parse(pageView[0].metadata).pageId;
+        expect(pageId).toEqual(expectedPage);
+        expect(navigationPageId).toEqual(expectedPage);
+
+        // Add all event ids to eventIds array
+        addEventsToList(parsedPayload, eventIds);
+    });
+
+    test('test FID and LCP events are emitted', async () => {
```

Comment:
question: What value do these tests add above the integration tests we currently have?

If we’re just checking that we get a 200 from DP, then I think one test is sufficient to cover it.

Later, when we verify that events were ingested, we will want to split up the tests in this manner.